source/features/ci-link.tsx

```diff
@@ -1,53 +1,51 @@
 import './ci-link.css';
+import React from 'dom-chef';
 import select from 'select-dom';
 import * as pageDetect from 'github-url-detection';
 
 import features from '.';
-import fetchDom from '../helpers/fetch-dom';
-import getDefaultBranch from '../github-helpers/get-default-branch';
-import {buildRepoURL, getCurrentCommittish} from '../github-helpers';
-
-// Look for the CI details dropdown in the latest 2 days of commits #2990
-const ciDetailsSelector = '.TimelineItem--condensed:nth-of-type(-n+2) batch-deferred-content[data-url$="checks-statuses-rollups"]';
-
-async function getCiDetails(): Promise<HTMLElement | undefined> {
-	if (pageDetect.isRepoHome()) {
```

Comment:
We should stop having multiple ways to fetch the same content, it makes it really difficult to debug it. We do this in a few places. Maybe the only place where it's worth it is the "current committish" situation but I'm not entirely sure about that either.